Source/WebGPU/WebGPU/Buffer.mm

```diff
@@ -85,7 +85,7 @@ static MTLStorageMode storageMode(bool deviceHasUnifiedMemory, WGPUBufferUsageFl
 {
     if (deviceHasUnifiedMemory)
         return MTLStorageModeShared;
-    if ((usage & WGPUBufferUsage_MapRead) || (usage & WGPUBufferUsage_MapWrite))
+    if ((usage & WGPUBufferUsage_MapRead) || (usage & WGPUBufferUsage_MapWrite) || (usage & WGPUBufferUsage_Index))
```

Comment:
something like:
```
if ((usage & (WGPUBufferUsage_MapRead | WGPUBufferUsage_MapWrite | WGPUBufferUsage_Index)) {
    ...
}
```
Would be easier for me to read.

Reply:
Good idea, I took that suggestion